internal/app/singularity/remote_get_login_password.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2018-2023, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+package singularity
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	scslibclient "github.com/sylabs/scs-library-client/client"
+	"github.com/sylabs/singularity/pkg/sylog"
+)
+
+var errNotSupported = errors.New("This library does not support direct OCI access")
+
+// RemoteGetLoginPassword retrieves cli token from oci library shim
+func RemoteGetLoginPassword(config *scslibclient.Config) (string, error) {
+	client := http.DefaultClient
```

Comment:
Please could you use an http.Client that has a timeout here, as this shouldn't be a long operation.

Reply:
implemented